README.rst

```diff
@@ -13,8 +13,9 @@ ExoCTK is an open-source, modular data analysis package focused primarily on atm
 * Integrations and Groups Calculator
 * Transit Light-Curve Fitter
 * Limb Darkening Calculator
-* Atmopsheric Retrievals
-* Atmospheric Forward Modeling - Currently only available through the `website <https://exoctk.stsci.edu/fortney>`_
+* Atmospheric Retrievals
+* Phase Constraint Calculator
+* Atmospheric Forward Modeling - Currently only available through the `website <https://exoctk.stsci.edu/fortney>`_.
```

Comment:
This is no longer true, actually. Why don't we just leave it as its own bullet point like the rest.